### ISSUE TITLE
Czr option to only "hide" the site title

### ISSUE THREAD

**Issue by mirucon** [mirucon/coldbox#8]
In the current version of Coldbox, v1.5.3, there's only able to set the site title to `display: none`.
It should have a customizer option to "hide" the site title for SEO and screenreader purposes.
